python/ray/experimental/channel/shared_memory_channel.py

```diff
@@ -19,6 +19,13 @@
 # _ResizeChannel class along with any metadata.
 MIN_BUFFER_SIZE = int(1000)  # 1000 bytes
 
+# We still need to add support for transferring objects that are larger than the gRPC
+# payload limit, which Ray sets to ~512 MiB (so we set it slightly lower here to be
+# safe).
+# TODO(jhumphri): Add support for transferring objects that are larger than the gRPC
+# payload limit. We can support this by breaking an object into multiple RPCs.
+GRPC_LIMIT = int(1024 * 1024 * 450)  # 450 MiB
```

Comment:
Can you move this into the config introduced by #46214?

Reply:
Done. I moved the constant to `context.py`, but I did not add a variable to the dataclass since the value cannot be configured.